src/model/immutable/BlockNode.js

```diff
@@ -29,6 +29,8 @@ export type BlockNodeConfig = {
   type?: DraftBlockType,
 };
 
+// https://github.com/facebook/draft-js/issues/1492
```

Comment:
Not sure why github is showing this as changes, this is already in master https://github.com/facebook/draft-js/blob/master/src/model/immutable/BlockNode.js#L32 and have been merged on https://github.com/facebook/draft-js/commit/652afc1a073702c351ad5be107c3febdcfe9936e